test/e2e/basic_workflow_test.go

```diff
@@ -77,12 +77,11 @@ func TestWrongCommand(t *testing.T) {
 	r := test.NewKnRunResultCollector(t, it)
 	defer r.DumpIfFailed()
 
-	out := test.Kn{}.Run("source", "apiserver", "noverb", "--tag=0.13")
```

Comment:
Seems with changes that flag validation is taking place, so error was coming back saying `--tag` is not a valid flag for `kn source apiserver noverb`. I removed `--tag` for now.